kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/AssertSoftlyTest.kt

```diff
@@ -254,6 +269,33 @@ class AssertSoftlyTest : FreeSpec({
             error.message shouldNotContain "3) "
          }
       }
+      "doesn't loose stack traces" - {
```

Comment:
```suggestion
      "doesn't lose stack traces" - {
```

Same at other change locations.